manifests/service.pp

```diff
@@ -4,30 +4,17 @@
 # @author Nick Markowski <nmarkowski@keywcorp.com>
 #
 class gdm::service {
-  if 'systemd' in $facts['init_systems'] {
-    $_services = [
```

Comment:
Don't these services need to be declared for svckill?

Reply:
@jeefberkey I 100% forgot about that and that's a good call :-|.

We should probably have acceptance tests that tie in svckill after checking the base.